src/Parser/DemonHunter/Vengeance/Modules/Abilities.js

```diff
@@ -162,6 +162,19 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 0.90,
+          extraSuggestion: combatant.hasTalent(SPELLS.FLAME_CRASH_TALENT.id)?<span>Line this up with <SpellLink id={SPELLS.INFERNAL_STRIKE.id} />to double stack <SpellLink id={SPELLS.SIGIL_OF_FLAME.id} /> because of the <SpellLink id={SPELLS.FLAME_CRASH_TALENT.id} /> talent.</span>:<span>Cast on cooldown for a dps increase.</span>,
```

Comment:
Please use the `React.Fragment` component instead of `span` for wrapping elements without special styling.

Reply:
Please pad your code with sufficient white-spacing to keep it readable, also there's a space missing after Infernal Strike.

Reply:
`Cast on cooldown for a dps increase.` should be a string, since there's no JSX in it.